SimPEG/electromagnetics/static/resistivity/simulation_2d.py

```diff
@@ -60,9 +62,13 @@ def g(k):
 
                 return phi, g
 
-            # find the minimum cell spacing, and the maximum side of the mesh
-            min_r = min(*[np.min(h) for h in self.mesh.h])
-            max_r = max(*[np.sum(h) for h in self.mesh.h])
+            if isinstance(self.mesh, discretize.CurvilinearMesh):
```

Comment:
Interesting are you using the CurvilinearMesh?

Reply:
Yes that is correct.